core/src/com/goonsquad/galactictd/systems/initialization/ScoreScreenInitSystem.java

```diff
@@ -1,16 +1,19 @@
 package com.goonsquad.galactictd.systems.initialization;
 
 import com.artemis.ComponentMapper;
+import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.math.Vector2;
 import com.goonsquad.galactictd.GalacticTDGame;
 import com.goonsquad.galactictd.components.graphics.Renderable;
+import com.goonsquad.galactictd.components.input.Event;
 import com.goonsquad.galactictd.components.input.Touchable;
 import com.goonsquad.galactictd.components.positional.MoveToPoint;
 import com.goonsquad.galactictd.components.positional.MovementDestination;
 import com.goonsquad.galactictd.components.positional.MovementSpeed;
 import com.goonsquad.galactictd.components.positional.Position;
 import com.goonsquad.galactictd.components.positional.ResetPosition;
+import com.goonsquad.galactictd.screens.HomeScreen;
```

Comment:
It is used [here](https://github.com/kevinfulkerson/towerdefense/pull/7/files#diff-1e610c7a9af7adc56ebe7403f801128dR55) to set the game to the home screen.